File: exercises/beecrowd/1095/1095.ts
// https://judge.beecrowd.com/en/problems/view/1095
function main(): void {
	let i = 1;
	let j = 60;

	while (j >= 0) {
		console.log(`I=${i} J=${j}`);
		i += 3;
		j -= 5;
	}
}

main();

File: exercises/beecrowd/1095/1095.java
// https://judge.beecrowd.com/pt/problems/view/1095
public class Main {
    public static void main(String[] args) {
        int I = 1;
        int J = 60;

        while (J >= 0) {
            System.out.println("I=" + I + " J=" + J);
            I += 3;
            J -= 5;
        }
    }
}
